### PR TITLE
fix(core): expose Util package to platform

### PR DESCRIPTION
**Pull request by Lokanathan-k**
need to extend the library for the UTIL package for the platform team to use the code instead of rewriting the same.
fixes: #1269

####Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/1269

#### Please provide a brief summary of this pull request.
need to extend the library for the UTIL package for the platform team to use the code instead of rewriting the same.

#### If this is a new feature, have you updated the documentation?
enhancement of existing feature

### CODE REVIEW

File: libs/core/src/index.ts [hunk 1]
```diff
@@ -43,4 +43,4 @@ export * from './lib/time-picker/public_api';
 export * from './lib/toggle/public_api';
 export * from './lib/token/public_api';
 export * from './lib/tree/public_api';
-
+export * from './lib/utils/pipes/pipe.module';
```
Comment (fkolar) [inline libs/core/src/index.ts]: You should add your export record to public_api not directly here

**Comment by fkolar** [comment]
https://github.com/SAP/fundamental-ngx/wiki/wip:-Contribute

File: libs/core/src/index.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import {angularCliVersion} from '@nrwl/schematics/src/lib-versions';
+import { angularCliVersion } from '@nrwl/schematics/src/lib-versions';
```
[the remark this answers is not in the document]
Comment (Lokanathan-k) [reply libs/core/src/index.ts]: @droshev i made the indentation changes and i am unable to commit the code i m seeing the following lint doc error. 
Screenshot 2019-10-17 at 8 44 04 AM

File: libs/core/src/index.ts [hunk 3]
```diff
@@ -43,4 +43,3 @@ export * from './lib/time-picker/public_api';
 export * from './lib/toggle/public_api';
 export * from './lib/token/public_api';
 export * from './lib/tree/public_api';
-
```
Comment (droshev) [inline libs/core/src/index.ts]: you can revert this file change

**Comment by droshev** [comment]
@Lokanathan-k can you update the title of the PR per https://github.com/SAP/fundamental-ngx/wiki/wip:-Contribute

**Comment by droshev** [comment]
@Lokanathan-k i updated it

**Pull request merged**